forgotPassword.php

```diff
@@ -0,0 +1,51 @@
+<?php
+    session_cache_expire(30);
+    session_start();
+    require_once('include/api.php');
+    ini_set("display_errors",1);
+    error_reporting(E_ALL);
+    $accessLevel = 0;
+    $userID = null;
+    if (isset($_SESSION['_id'])) {
+        // 0 = not logged in, 1 = standard user, 2 = manager (Admin), 3 super admin (TBI)
+        $accessLevel = $_SESSION['access_level'];
+        $userID = $_SESSION['_id'];
+    }
+
+    if ($_SERVER["REQUEST_METHOD"] == "POST") {
+        require_once('include/input-validation.php');
+        require_once('domain/Person.php');
+        require_once('database/dbPersons.php');
+        if (!wereRequiredFieldsSubmitted($_POST, array('password', 'new-password'))) {
+            echo "Args missing";
+            die();
+        }
+        $newPassword = $_POST['new-password'];
+        $user = retrieve_person($userID);
```

Comment:
So since the user won't be logged in yet, the $userID won't be set and will be null, so I think retrieve_person and change_password() on line 26 won't work

Reply:
Currently, it won't do anything but also won't produce any errors. I kept it in because I thought it could be used later since it will be needed when changing their password. 
Even though the user hasn't logged in yet, when they enter their email in the verifyEmail.php page, it can be used to get their ID which will then later be used when changing their password in forgotPassword.php. I can remove those lines for now if you think that would be best.

Reply:
Oh ok just wanted to point it out but if you see it then shouldn't be an issue